tests/foreman/longrun/test_oscap.py

```diff
@@ -315,24 +295,9 @@ def test_positive_oscap_run_via_ansible_bz_1814988(
             'organizations': module_org.name,
         }
     )
-    with Broker(
-        nick='rhel7',
-        host_class=ContentHost,
-        target_cores=OSCAP_TARGET_CORES,
-        target_memory=OSCAP_TARGET_MEMORY,
-    ) as vm:
-        host_name, _, host_domain = vm.hostname.partition('.')
-        vm.install_katello_ca(target_sat)
-        vm.register_contenthost(module_org.name, ak_name['rhel7'])
-        assert vm.subscribed
-        Host.set_parameter(
```

Comment:
Why `Host.set_parameter` is removed?

Reply:
I don't think it will affect, the rex job works fine without switching the `remote_execution_connect_by_ip`parameter, I don't see this being used elsewhere.

Reply:
It will work fine if fqdn is resolved but if we want to connect by ip,  which in few test cases we do. I guess keeping this won't harm. Also `connect_by_ip` setting is by default set to False.

Reply:
yes, REX can be run on hostname, connect by ip is only required for provisioned hosts where hostnames will be unreachable